### ISSUE TITLE
Cancel and Upload buttons are not marked as focused in Upload widget

### ISSUE THREAD

**Issue by nelito987** [telerik/kendo-ui-core#6001]
### Bug report

In some of the SASS themes when a file is uploaded, 'Cancel' and 'Upload' buttons are not marked as focused when navigating through Tab key

### Reproduction of the problem
1. Open the Upload [Overview Demo](https://demos.telerik.com/kendo-ui/upload/index)
2. Select one of the following themes - Fiori, Flat, Metro, Metro Black, Nova, Office 365
3. Upload a file
4. Navigate by using the Tab key

### Current behavior
The 'Upload' and 'Cancel' buttons are not styled as focused

### Expected/desired behavior
The 'Upload' and 'Cancel' buttons should be styled as focused

### Environment

* **Kendo UI version:** 2020.2.617
* **Browser:** [all]
